Tests/scripts/validate_files_structure.py

```diff
@@ -631,6 +640,8 @@ def validate_modified_files(integration_set, modified_files, playbook_set, scrip
     has_schema_problem = False
     for file_path in modified_files:
         print "Validating {}".format(file_path)
+        if "conf.json" in file_path:
```

Comment:
perhaps like in secrets we should have an array of skipped files, for readability purposes.